spec/cell_spec.rb

```diff
@@ -50,21 +46,37 @@
       expect(@cell.empty?).to eq(false)
     end
   end
+  
+  describe "#fired_upon?" do
+    it 'will return state of fired_upon attribute' do
+      expect(@cell.fired_upon?).to eq(false)
+    end
+  end
 
   describe "#fire_upon" do
     it 'will update the status of fired_upon attribute' do
+      expect(@cell.fired_upon).to eq(false)
+      @cell.fire_upon
 
+      expect(@cell.fired_upon).to eq(true)
     end
-
+    
     it 'will NOT revert status of fired_upon from true to false' do
+      @cell.fire_upon
+      expect(@cell.fired_upon).to eq(true)
+      @cell.fire_upon
 
+      expect(@cell.fired_upon).not_to eq(false)
```

Comment:
Could we maybe change this test wording? I like the to eq and haven't really used the not to eq wording. I am open to this, I would like to talk about it.

Reply:
Oops just wanted to leave a single comment.